### DISCUSSION TITLE
I can't install again.

### DISCUSSION THREAD

**Discussion by Joel-barros** [skyline-emu/skyline#1621] [Q&A] [answered]
Tentei atualizar o app mas deu o erro "Como o pacote tem um conflito com um pacote já existente, o app não foi instalado.", então desinstalei o aplicativo antigo e tentei instalar a nova versão mas não tive sucesso. Ainda aparece a mensagem de erro "Como o pacote tem um conflito com um pacote já existente, o app não foi instalado."

**Comment by yhak22** [comment, marked as answer]
1- copy your save data from (android / data /skyline.emu)
2- remove the old app
3- then install the new one 
4- put ur saved data in the skyline folder
